api/krusty/fnplugin_test.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 func TestFnExecGenerator(t *testing.T) {
-	th := kusttest_test.MakeEnhancedHarness(t)
-	defer th.Reset()
+	// Function plugins should not need the env setup done by MakeEnhancedHarness
+	th := kusttest_test.MakeHarness(t)
```

Comment:
This is a weird way to regression test, but it does reproduce the issue on master. I was trying (unsuccessfully) to foil plugin root discovery by getting rid of the possible sources when I realized I was essentially just undoing HarnessEnhanced, and it would be equivalent to just not use it AFAICT. Lmk if you have a better way.

Reply:
it's fine.  feel free to improve the testing harness in any way.  it has evolved organically to fill testing needs